.github/actions/yarn-install-and-build/action.yml

```diff
@@ -11,19 +11,21 @@ inputs:
   build_core:
     description: If not false it will build the core package
     required: false
-    default: true
+    default: 'true'
```

Comment:
Why do these need quotes?

Reply:
YAML stuff. I think github will cast it to a string anyways, but the inputs have to be strings I think

Reply:
Why only these inputs and not all of them?

Reply:
it is all of them. for actions at least I think? tbh I'm just doing what my IDE linter tells me to do.